lib/rubocop/config.rb

```diff
@@ -411,9 +411,12 @@ def warn_about_unrecognized_cops(invalid_cop_names)
       end
     end
 
-    def validate_syntax_cop(valid_cop_names)
-      return unless valid_cop_names.include?('Lint/Syntax') ||
-                    valid_cop_names.include?('Syntax')
+    def validate_syntax_cop
+      syntax_config = self['Lint/Syntax'] || self['Syntax']
```

Comment:
This reminds me that at some point we drop support for non-namespaced cop references. That's just a remnant of the past.